test/features/auth/sign_out_test.rb

```diff
@@ -0,0 +1,15 @@
+require "test_helper"
+
+feature "User Signs Out" do
+  scenario "User informed of log out" do
+    #visit user_sign_in
+    visit root_path
+    click_on 'Login'
+
+    fill_in "Email", with: users(:user_1).email
+    fill_in "Password", with: "password"
+    click_button "Log in"
```

Comment:
Lines 6 through 11 can be refactored into a method in the test_helper file, since they will be used regularly.

Something like `def sign_in(user)` would let us log in different users in different situations